tools/autograd/derivatives.yaml

```diff
@@ -2529,12 +2532,15 @@
 # fft
 - name: _fft_r2c(Tensor self, int[] dim, int normalization, bool onesided) -> Tensor
   self: fft_r2c_backward(grad, dim, normalization, onesided, self.size(dim.back()))
+  result: _fft_r2c(self_t, dim, normalization, onesided)
```

Comment:
If you find yourself just replacing `self` with `self_t` you can `auto_linear` macro for this!
It can help improve readability